lms/static/sass/course/_info.scss

```diff
@@ -56,6 +61,85 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    .upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      border-style: solid;
+      border-color: $uxpl-gray-background $uxpl-gray-background $uxpl-gray-background $uxpl-green-base;
+      border-width: 1px 1px 1px 10px;
```

Comment:
Rule of thumb is to watch out for any time you treat the "left" and "right" attributes differently.